app/src/main/java/com/benoitquenaudon/daggerbundleinjectiondemo/ui/BaseAppCompatActivity.java

```diff
@@ -4,21 +4,26 @@
 import android.support.annotation.Nullable;
 import com.benoitquenaudon.daggerbundleinjectiondemo.BundleService;
 import dagger.android.support.DaggerAppCompatActivity;
+import javax.inject.Inject;
 
 public abstract class BaseAppCompatActivity extends DaggerAppCompatActivity {
-  private BundleService bundleService;
+  @Inject BundleService bundleService;
+  private Bundle savedInstanceState;
```

Comment:
Is this field necessary? What did you feel needed improvement in the previous implementation?